### PR TITLE
API examples in step-by-step guide

### PR DESCRIPTION
**Pull request by daavoo**

### CODE REVIEW

File: docs/step-by-step-guide.md [hunk 1]
```diff
@@ -48,6 +48,33 @@ Cleaner input data ensures that the model works with reliable and consistent inf
 
    - Ensures the document is clean and ready for the next step.
 
+### 🔍 **API Example**
```
Comment (daavoo) [inline docs/step-by-step-guide.md:51]: @stefanfrench not sure if this is the best format but I put a quick example to test how it would look

**Comment by stefanfrench** [comment]
- Updated the step-by-step guide to include API examples
- Updated customization guide to reflect latest changes
- Added contributions to read.me to be consistent with template

File: docs/customization.md
Comment (daavoo) [inline docs/customization.md:1]: Awesome! I was about to say that we needed to update this for the CLI 🚀

**Review by Kostis-S-Z — APPROVED** [review]
:cool:

**Pull request merged**